lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1956,6 +1961,11 @@ definitions:
       nat_enabled:
         type: boolean
         example: true
+      enable_converged_core:
```

Comment:
can you document how 'enable_converged_core' value used when gateway and epc level values differs.

Reply:
Sure, I will document on it and will update soon.